getting_started/6_Understanding_Qubits_and_Quantum_Gates.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<blockquote>There are 2 conditions for quantum gates - <em>Unitary</em> ( preserves length) and <em>Reversible</em></blockquote><p>Not <em>quite</em> accurate; and a bit redundant; unitarity guarantees that an operator is both an isometry (preserves length) and surjective (and thus bijective and thus reversible).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Actually, since we're talking operators, even the surjective part isn't necessary; unitarity implies isometry implies injectivity implies bijectivity. Still, this would be better worded as "A quantum gate has to be _unitary_, that is, it must preserve distances and be reversible" or something along those lines.

Reply:
updated